flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/utils/ConfigurationUtils.java

```diff
@@ -48,4 +52,11 @@ public static Configuration loadMapFormattedConfig(Path configPath) throws Excep
                     e);
         }
     }
+
+    public static boolean isDeploymentMode(CommandLine commandLine) {
+        String target = commandLine.getOptionValue(TARGET);
+        return target != null
+                && !target.equalsIgnoreCase("local")
+                && !target.equalsIgnoreCase("remote");
```

Comment:
Can we add a enum to maintain these variables, add provide `isDeploymentMode` method?